classes/rental.rb

```diff
@@ -0,0 +1,11 @@
+# Rental class
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Great job so far, in the rental we need to create a `belongs-to` relationship between rental and book, and also rental and person.
`    book.rental << self`